onboarder/installation/index.rst

```diff
@@ -0,0 +1,166 @@
+Installation
+============
+
+.. warning::
+
+    Before starting the installation process, make sure you've fulfilled all the :doc:`prerequisites </onboarder/prerequisites/index>`
+
+Add the extension dependency to your PIM
+----------------------------------------
+
+Execute the following composer commands to require the bundle:
+
+.. code-block:: bash
+
+    composer config repositories.onboarder '{"type": "vcs", "url": "ssh://git@distribution.akeneo.com:443/pim-onboarder", "branch": "master"}'
+    composer config repositories.simplesamlphp '{"type": "vcs", "url": "ssh://git@distribution.akeneo.com:443/simplesamlphp-module-pimonboarder", "branch": "master"}'
+    composer require "akeneo/pim-onboarder" "1.1.*"
+
+
+Enable the extension
+--------------------
+
+Register the following two new bundle in your ``app/AppKernel.php``
+
+.. code-block:: php
+
+    protected function registerProjectBundles()
+    {
+        return [
+            // your app bundles should be registered here
+            new Pim\Onboarder\Bundle\PimOnboarderBundle(),
+            new Hslavich\OneloginSamlBundle\HslavichOneloginSamlBundle(),
+        ];
+    }
+
+Configure the extension
+-----------------------
+
+Modify your security configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SSO authentication process works quite differently than the classic login form method, it needs specific firewall and access control configuration.
+
+First, modify the configuration of the main firewall as follows :
+
+.. code-block:: yaml
+
+    main:
+        pattern:                        ^/
+        provider:                       chain_provider
+        saml:
+            username_attribute: 'uid'
+            check_path: '/saml/acs'
+            login_path: '/saml/login'
+        logout:
+            path:                       '/saml/logout'
+        anonymous:                      true
+
+Then modify the access control rules :
+
+.. code-block:: yaml
+
+    access_control:
+        - { path: ^/admin/, role: ROLE_ADMIN }
+        - { path: ^/api/rest/v1$, role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/api/, role: pim_api_overall_access }
+        #Additionnal access control for SSO
+        - { path: ^/user/login, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/saml-idp/resume, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/user/reset-request, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/user/send-email, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/saml/login, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/saml/metadata, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/, roles: IS_AUTHENTICATED_REMEMBERED }
+
+.. note::
+    If you had specific access controls needed by some other extension or customization, you can keep it. But please note that it can be imcompatible with this new setting and that for Symfony, the order is important (the first rule matched is applied).
```

Comment:
I don't understand `it can be imcompatible with this new setting and that for Symfony`